src/App.js

```diff
@@ -9,6 +9,26 @@ import {
 import AppHeader from './components/AppHeader';
 import Home from './pages/Home';
 import PostsManager from './pages/PostsManager';
+import { init, AuthType, LiveboardEmbed, EmbedEvent } from '@thoughtspot/visual-embed-sdk';
+import PostsList from './pages/PostsList';
+
+
+init({
+    thoughtSpotHost: "https://embed-1-do-not-delete.thoughtspotdev.cloud",
```

Comment:
We should not hardcode the TS host, it should be passed from the `process.env` environment variables when invoking the dev server